neurom/view/view.py

```diff
@@ -188,13 +190,33 @@ def plot_soma(ax, soma, plane='xy',
                                    ignore=False)
 
 
+def _get_isec_map(nrn, nrn_secid):
+    """Get NEURON isec from nrn morphology."""
+    from morph_tool import nrnhines
+    import tempfile
+
+    with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
Do you need the directory? A NamedTemporaryFile would not do as well?

Reply:
Ah good point, I think I did it this way after a very quick googling ;) I'll update!